crates/sc-proof-of-time/src/source.rs

```diff
@@ -230,6 +232,18 @@ where
             checkpoints,
         } = proof;
 
+        if self.sync_oracle.is_major_syncing() {
```

Comment:
Looks like time keeper is still producing proofs when syncing, this change only prevents the locally produced proofs from being gossiped? If so, why produce proofs when syncing?

Reply:
There is quite literally no reason, but making it pause and resume depending on sync status will introduce additional logic in the hot loop, so I though it would be nice to avoid that and let it run for no reason. Do you think it is justified?

Reply:
I think once we have infrastructure for dealing with standby timekeeper this will be easy to solve, but not right now

Reply:
In the prev implementation, the timekeeper loop would only start after the sync is completed. Is that not sufficient? (i.e) do we need to worry about stop/resume for sync that happen later?

Reply:
We kind of do. When node starts and PoT source is initialized node is not in major syncing mode, for it to enter major syncing it needs to connect to the network, exchange information about the tip of the chain and start syncing there.